tools/ctl/command/runbenchmark/runbenchmark.go

```diff
@@ -126,12 +196,31 @@ func Handler(ctx context.Context, cfg Config, deps *Deps) (*act.NoOutput, error)
 		if err != nil {
 			return nil, errors.Wrap(err, "Failed to create temp directory")
 		}
-		// TODO: Validate this.
-		prebuildURL := fmt.Sprintf("https://%s.storage.googleapis.com/%s", cfg.BootstrapBucket, cfg.BootstrapVersion)
+		var prebuildURL string
+		var needHostGate bool
+		if cfg.BootstrapBucket != "" && cfg.BootstrapVersion != "" {
+			prebuildURL = fmt.Sprintf("http://%s.storage.googleapis.com/%s", cfg.BootstrapBucket, cfg.BootstrapVersion)
+		} else {
+			fs := memfs.New()
```

Comment:
i'm not sure about the default behavior being attempting to do a local go build. this sort of implies that --local will always be run from a local checkout of the repo which i'm not sure we want to add as an assumption here. to me, --local is about execution (i.e. "execute on this host") and is less to do with being a sort of "development mode."

as such, having some explicit opt-in for users to tell the CLI how to get bootstrap tools feels like the right design. having a local source path from which we can build is perfectly fine. we could also support building from our module directly i.e. github.com/google/oss-rebuild/cmd/timewarp@latest.